dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/TestCase.mo

```diff
@@ -0,0 +1,144 @@
+within Dynawo.Examples.Nordic;
+
+  /*
+  * Copyright (c) 2022, RTE (http://www.rte-france.com)
+  * See AUTHORS.txt
+  * All rights reserved.
+  * This Source Code Form is subject to the terms of the Mozilla Public
+  * License, v. 2.0. If a copy of the MPL was not distributed with this
+  * file, you can obtain one at http://mozilla.org/MPL/2.0/.
+  * SPDX-License-Identifier: MPL-2.0
+  *
+  * This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+  */
+
```

Comment:
When running the case I have the following error : ##IDA## linear solver failed try once again = 1.00256074400055

Reply:
For more information: The testcase runs successfully on my windows machine with OpenModelica 1.18.1/OMSimulator v2.1.1.post80-g1bf17f4-mingw and the old front-end and the version 3.2.3 of the modelica package, using euler or IDA, with the solver settings in the annotations (interval 0.01, tolerance 0.005)

Reply:
On my machine, with OMCompiler v1.17.0 and OMSimulator v2.0.0.post284-gc8ec782-linux, the simulation of TestCase.mo stops at t = 170.913 s. The following message can be read in the monitor :

The initialization finished successfully without homotopy method.
Dynawo.Examples.Nordic.TestCase_16dae.c:20148: Invalid root: (-5.74863)^(6.0257)
Dynawo.Examples.Nordic.TestCase_16dae.c:20148: Invalid root: (-5.74908)^(6.0257)
##IDA## discrete update failed flag -8!
Simulation process failed. Exited with code 255.

Reply:
If I put the terminals before the components in TransformerWithControl.mo (as per the coding style), the simulation stops at t = 167.249 s with the following message :

The initialization finished successfully without homotopy method.
Dynawo.Examples.Nordic.TestCase_16dae.c:29933: Invalid root: (-0.0464225)^(6.0257)
##IDA## -8 error occurred at time = 167.42861443606
model terminate | Integrator failed. | Simulation terminated at time 167.429
Simulation process failed. Exited with code 247.